### PR TITLE
Certificate validation

### PR DESCRIPTION
**Pull request by cbardini**

### CODE REVIEW

File: test_config.placeholder.json [hunk 1]
```diff
@@ -1,9 +1,14 @@
 {
```
Comment (dcbroad3) [inline test_config.placeholder.json:1]: I mainly just have the same comments on this file as in the PIWebAPI PR. 
- Casing?
- Any reason to use an array?
- `endpoint-type` seems unnecessary?

Comment (dcbroad3) [reply test_config.placeholder.json:1]: @cbardini I think the only comment I still had on this is that "EndpointType" probably isn't necessary here.

Comment (cbardini) [reply test_config.placeholder.json:1]: @dcbroad3 I removed it from the README, but I forgot to remove it from the actual file. It should be good now.

**Review by derekendres — APPROVED** [review]
I would comment in history that we changed config file type

**Review by dcbroad3 — APPROVED** [review]

**Pull request merged**